tools/mk/Makefile.node.targ

```diff
@@ -38,7 +39,7 @@ else
 $(NODE_EXEC) $(NPM_EXEC) $(NODE_WAF_EXEC):
 	(mkdir -p $(BUILD) \
 		&& cd $(BUILD) \
-		&& [[ -d src-node ]] && (cd src-node && git checkout master && git pull) || git clone https://github.com/joyent/node.git src-node \
+		&& [[ -d src-node ]] && (cd src-node && git checkout master && git pull) || git clone https://github.com/nodejs/node.git src-node \
```

Comment:
I'm pretty sure that the branch here will need to be main. I had to fix that in the sdcnode repo. This also makes me wonder if this bit of code even ever gets used anymore...